macq/extract/slaf.py

```diff
@@ -1,16 +1,11 @@
 """.. include:: ../../docs/templates/extract/slaf.md"""
-
 from typing import Set, Union
-
 from bauhaus import Encoding
 from nnf import And, Or, Var, config, false, true
-
 import macq.extract as extract
-
-from ..observation import AtomicPartialObservation, ObservedTraceList
-from .exceptions import IncompatibleObservationToken
-from .learned_fluent import LearnedFluent
-from .model import Model
+from macq.observation import AtomicPartialObservation, ObservedTraceList
+from macq.extract.exceptions import IncompatibleObservationToken
+from macq.extract.model import Model
```

Comment:
Can you explain why this change went in? I'd like to try and fix the CI/CD, and I think that will include bumping all the python versions, but this change in the PR doesn't follow the pattern used by other extraction techniques (imports are relative). If you want to remove the changes on this file, then I can merge the rest and get started on fixing things off the main branch.

Thanks!

Reply:
i think i made those changes trying to fix circular imports error, because of the unsuccessful tests.

Reply:
i deleted the changes and added the delted lines in the file.